app/src/organisms/ODD/CameraSettings/CameraControls/index.tsx

```diff
@@ -21,21 +24,30 @@ export type ActiveControlView =
 
 export interface CameraControlsProps {
   toggleShowControls: () => void
+  runId?: string
```

Comment:
```suggestion
  runId: string | null
```
See comments in `CameraSettings/index.tsx`.